oci/private/container.bzl

```
@@ -3,102 +3,79 @@ _attrs = {
     "base": attr.string(
         mandatory = True,
     ),
-
     # See: https://github.com/opencontainers/image-spec/blob/main/config.md#properties
     "entrypoint": attr.string_list(),
     "cmd": attr.string_list(),
     "labels": attr.string_list(),
     "tag": attr.string_list(),
     "layers": attr.label_list(),
+    "_container_sh_tpl": attr.label(default = "container.sh.tpl", allow_single_file = True),
 }
 
-def _strip_external(path):
-    return path[len("external/"):] if path.startswith("external/") else path
-
 def _impl(ctx):
-    toolchain = ctx.toolchains["@contrib_rules_oci//oci:crane_toolchain_type"]
-
-    launcher = ctx.actions.declare_file("crane.sh")
+    crane = ctx.toolchains["@contrib_rules_oci//oci:crane_toolchain_type"]
+    registry = ctx.toolchains["@contrib_rules_oci//oci:registry_toolchain_type"]
 
-    # TODO: dynamically get --platform from toolchain
-    ctx.actions.write(
-        launcher,
-        """#!/usr/bin/env bash
-set -euo pipefail
-{crane} $@""".format(
-            crane = toolchain.crane_info.crane_path,
-        ),
+    launcher = ctx.actions.declare_file("container_%s.sh" % ctx.label.name)
```

Nit: the preferred style in Python (and Starlark being a subset is included) is the more modern `format`.
```suggestion
    launcher = ctx.actions.declare_file("container_{}.sh".format(ctx.label.name))
```

---

feels like we should have a ticket in the SIG for improving buildifier to have linter for this